docs/FAQ.md

```diff
@@ -0,0 +1,52 @@
+<!-- Copyright (c) Microsoft Corporation. All rights reserved.
+     Licensed under the MIT License. -->
+     
+     ## FAQ
```

Comment:
```suggestion
## FAQ
```
The header markup isn't kicking in due to the space between the start of the line and the markup